designs/usb-light.stanza

```diff
@@ -0,0 +1,45 @@
+#use-added-syntax(jitx)
+defpackage ocdb/designs/usb-light :
+  import core
+  import collections
+  import jitx
+  import jitx/commands
+  import ocdb/defaults
+  import ocdb/generic-components
+  import ocdb/bundles
+  import ocdb/design-vars
+  import ocdb/micro-controllers
+  import ocdb/tolerance
+  import ocdb/symbols
+  import ocdb/property-structs
+
+val BOARD-SHAPE = RoundedRectangle(25.0, 25.0, 0.25)
+pcb-module main-board:
+  inst conn : ocdb/amphenol/10103594-0001LF/component
+  port vbus : power
+  property(conn.VCC.power-pin) = PowerPin(Interval(4.75, 5.5, 5.0))
+  inst led : gen-led-cmp(tol(2.1,0.1), 0.02, "YELLOW", "0805")[8]
+  net p5v (vbus.vdd conn.VCC)
+  net gnd (vbus.gnd conn.GND conn.SHIELD)
+  symbol(p5v) = ocdb/symbols/supply-sym
+  symbol(gnd) = ocdb/symbols/ground-sym
+  val vbus-voltage = (nominal(recommended-voltage(property(conn.VCC.power-pin)))) as Double
+  val resistor-voltage = vbus-voltage - (2.0 * property(led[0].forward-voltage))
+  println("Vbus = %_" % [vbus-voltage])
+  println("Vf = %_" % [property(led[0].forward-voltage)])
+  val exact-resistance = resistor-voltage / property(led[0].test-current)
```

Comment:
You should probably isolate the maths in a function

Reply:
created function led-circuit-resistance (